docs/src/contributing_apub_api_outline.md

```diff
@@ -1,380 +1,592 @@
-```
-{
-  "@context": "https://www.w3.org/ns/activitystreams",
-  "type": "Delete",
-  "actor": id,
-  "object": comment_id, or post_id
-}
-```
-#### [Update](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-update)
-```
-{
-  "@context": "https://www.w3.org/ns/activitystreams",
-  "type": "Create",
-  "actor": id,
-  "object": comment_id, or post_id
-  "content": "New comment",
-  "updated": "New Date"
-}
-```
-#### [Read](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-read)
-```
-{
-  "@context": "https://www.w3.org/ns/activitystreams",
-  "type": "Read",
-  "actor": user_id
-  "object": comment_id
-}
-```
+| Field Name | Mandatory | Description |
+|---|---|---|
+| `attributedTo` | yes | ID of the user which created this post |
+| `to` | yes | ID of the community where it was posted to |
+| `summary` | yes | Title of the post |
+| `content` | no | Body of the post |
+| `url` | no | An arbitrary link to be shared |
+| `image` | no | Thumbnail for `url`, only present if it is an image link |
+| `commentsEnabled` | yes | False indicates that the post is locked, and no comments can be added |
+| `sensitive` | yes | True marks the post as NSFW, blurs the thumbnail and hides it from users with NSFW settign disabled |
+| `stickied` | yes | True means that it is shown on top of the community |
+| `published` | no | Datetime when the post was created |
+| `updated` | no | Datetime when the post was edited (not present if it was never edited) |
 
-#### [Like](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-like)
-- TODO: Should likes be notifications? IE, have a to?
-```
-{
-  "@context": "https://www.w3.org/ns/activitystreams",
-  "type": "Like",
-  "actor": user_id
-  "object": comment_id
-  // TODO different types of reactions, or no?
-}
-```
-#### [Dislike](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-dislike)
-```
-{
-  "@context": "https://www.w3.org/ns/activitystreams",
-  "type": "Dislike",
-  "actor": user_id
-  "object": comment_id
-  // TODO different types of reactions, or no?
-}
-```
+### Comment
 
-### Posts
-#### [Create](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-create)
-```
+A reply to a post, or reply to another comment. Contains only text (including references to other users or communities). Lemmy displays comments in a tree structure.
+
+```json
 {
-  "@context": "https://www.w3.org/ns/activitystreams",
-  "type": "Create",
-  "actor": id,
-  "to": community_id/followers
-  "object": post_id
+    "@context": "https://www.w3.org/ns/activitystreams",
+    "id": "https://enterprise.lemmy.ml/comment/95",
+    "type": "Note",
+    "attributedTo": "https://enterprise.lemmy.ml/u/picard",
+    "to": "https://enterprise.lemmy.ml/c/main",
+    "content": "mmmk",
+    "inReplyTo": [
+        "https://enterprise.lemmy.ml/post/38",
+        "https://voyager.lemmy.ml/comment/73"
+    ],
+    "published": "2020-10-06T17:53:22.174836+00:00",
+    "updated": "2020-10-06T17:53:22.174836+00:00"
```

Comment:
We don't need to change this here right now, but I think this is where you suggested we add a to / CC for the parent comment actor_id. Maybe add a TODO

Reply:
Right, that was in the comment create/update activity. I added it to #698.